docs/5.x/quickstart.md

```diff
@@ -1,231 +1,373 @@
+-------------------|---------------------------------
+`--token`          | A secret token of your choosing which will be used to add additional nodes to this cluster in the future. We'll use word "secret" here.
+`--advertise-addr` | The IP address this host will be visible on by other nodes in this cluster. We'll use `10.5.5.28`.
 
-As shown in the [Overview](overview.md), there are two ways to install an
-Application Bundle packaged with Gravity:
+The command below will create a single-node Kubernetes cluster with Mattermost running inside:
 
-* Online, via the Ops Center.
-* Offline, via a downloadable tarball.
+```
+# We are executing this on the node named 'host' with IP address of 10.5.5.28
+$ sudo ./gravity install \
+        --advertise-addr=10.5.5.28 \
+        --token=secret
+# Output:
+Sat Jan 12 05:30:44 UTC Starting installer
+Sat Jan 12 05:30:44 UTC Preparing for installation...
+Sat Jan 12 05:31:09 UTC Installing application mattermost:2.2.0
+Sat Jan 12 05:31:09 UTC Starting non-interactive install
+Fri Jan 11 21:31:09 UTC Auto-loaded kernel module: br_netfilter
+Fri Jan 11 21:31:09 UTC Auto-loaded kernel module: iptable_nat
+Fri Jan 11 21:31:09 UTC Auto-loaded kernel module: iptable_filter
+Fri Jan 11 21:31:09 UTC Auto-loaded kernel module: ebtables
+Fri Jan 11 21:31:09 UTC Auto-set kernel parameter: net.ipv4.ip_forward=1
+Fri Jan 11 21:31:09 UTC Auto-set kernel parameter: net.bridge.bridge-nf-call-iptables=1
+Sat Jan 12 05:31:10 UTC All agents have connected!
+Sat Jan 12 05:31:10 UTC Starting the installation
+Sat Jan 12 05:31:11 UTC Operation has been created
+Sat Jan 12 05:31:12 UTC Execute preflight checks
+Sat Jan 12 05:31:16 UTC Configure packages for all nodes
+Sat Jan 12 05:31:20 UTC Bootstrap master node host
+Sat Jan 12 05:31:24 UTC Pull packages on master node host
+Sat Jan 12 05:32:20 UTC Install system package teleport:2.4.7 on master node host
+Sat Jan 12 05:32:22 UTC Install system package planet:5.2.19-11105 on master node host
+Sat Jan 12 05:32:48 UTC Wait for system services to start on all nodes
+Sat Jan 12 05:33:24 UTC Label and taint master node host
+Sat Jan 12 05:33:25 UTC Bootstrap Kubernetes roles and PSPs
+Sat Jan 12 05:33:26 UTC Export applications layers to Docker registries
+Sat Jan 12 05:33:27 UTC Populate Docker registry on master node host
+Sat Jan 12 05:34:24 UTC Install system application dns-app:0.1.0
+Sat Jan 12 05:34:31 UTC Install system application logging-app:5.0.2
+Sat Jan 12 05:35:05 UTC Install system application tiller-app:5.2.1
+Sat Jan 12 05:35:43 UTC Install system application site:5.2.4
+Sat Jan 12 05:36:59 UTC Install system application kubernetes:5.2.4
+Sat Jan 12 05:37:00 UTC Install application mattermost:2.2.0
+Sat Jan 12 05:37:10 UTC Enable elections
+Sat Jan 12 05:37:12 UTC Operation has completed
+Sat Jan 12 05:37:13 UTC Installation succeeded in 6m3.257480586s
+```
 
-### Online Installer
+**Congratulations!** You have created a fully functional, highly available
```

Comment:
The command above installed a single-node cluster so I would remove "highly available".